src/darwin/Framework/CHIP/CHIPCommissioningParameters.m

```diff
@@ -0,0 +1,26 @@
+/**
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import "CHIPCommissioningParameters.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation CHIPCommissioningParameters : NSObject
+
```

Comment:
Shouldn't we init the members to nil when we are inited?  Or does that happen automatically?

Reply:
They are automatically nil.